less/c2corg_ui.less

```diff
@@ -380,6 +380,7 @@ div.help-block {
     opacity: 0;
     pointer-events: none;
     width: 100%;
+    height: ~"-webkit-calc(100% - 120px)";
```

Comment:
Safari is based upon Webkit, right? Then it's probably concerned by this change.
Chrome is no longer using it since 2013 (OK it uses a fork I think), then will this ``-webkit-calc`` really impacts Chrome?

Reply:
I honestly don't know much about Safari
As I said i've tried it using the lastest chrome version on my android.
https://developers.google.com/web/tools/chrome-devtools/remote-debugging/

Adding this in the CSS results in fixing the bug